src/renderer/src/components/Contact/ContactList.tsx

```diff
@@ -16,6 +17,7 @@ type TProps = {
   onAddressSelected?: (address: TContactAddress | null) => void
   selectFirst: boolean
   showSelectedIcon: boolean
+  tokenFilter?: TokenBalance | null
```

Comment:
It would be better to rename it to blockchainFilter, it will turn this component more reusable.
```suggestion
  blockchainFilter?: TBlockchainServiceKey
```